flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/catalog/MySql56CatalogITCase.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.connector.jdbc.catalog;
+
+import org.apache.flink.connector.jdbc.test.DockerImageVersions;
+
+import org.testcontainers.containers.MySQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+/** E2E test for {@link MySqlCatalog}. */
+@Testcontainers
+public class MySql56CatalogITCase extends MySqlCatalogTestBase {
+
+    @Container
+    private static final MySQLContainer<?> CONTAINER =
+            createContainer(DockerImageVersions.MYSQL_5_6);
+
+    @Override
+    protected String getDatabaseUrl() {
+        return CONTAINER.getJdbcUrl().substring(0, CONTAINER.getJdbcUrl().lastIndexOf("/"));
+    }
```

Comment:
It looks like this method has same implementation for 
`MySql57CatalogITCase`, `MySql56CatalogITCase`, `MySqlCatalogITCase`.
Does it make sense to move this implementation on `MySqlCatalogTestBase` level?

Reply:
Makes sense, but I will say: Its really needed test this against 3 versions of MySql?